user-manual/administer/settings.rst

```diff
@@ -1871,3 +1872,95 @@ setting.
+
+None of the information gathered through the use of cookies or Google Analytics is used for any purpose other than the ones described here.
+
+System administrators will need to perform some manual steps to finalize the setup of the new features when upgrading or installing AtoM 2.4.1 for the first time.
+
+New 2.4.1 installations
+----------------------------
+
+Update cookie banner
+^^^^^^^^^^^^^^^^^^^^
+
+* The default cookie notification banner message will require updating.
+  This can be done in ''Admin > Settings > Privacy Notification''.
+* The default message will require the URL to the Privacy Policy static page
+  be updated. Currently the message reads:
+
+<pre>
+"This website uses cookies to enhance your ability to browse and load content. This website uses cookies to enhance your ability to browse and load content. More Info: http://10.10.10.10/privacy"
+</pre>
+
+Administrators can edit the text of the banner message, and choose whether or not to display the banner when a user accesses your site for the first time. Here's an example of how the notification banner will look:
+
+.. image:: images/gdpr-banner-post-set.*
+   :align: center
+   :width: 80%
+   :alt: GDPR banner first access
+
+* Update the URL in the message to the base URL for your site
+  (see :ref: `Site information <site-information>`)
+* If desired, you can use the current custom link syntax formatting to style
+  the hyperlink -  see the `AtoM formatting documentation <https://www.accesstomemory.org/docs/latest/user-manual/add-edit-content/formatting/>` for more information
+
+Edit the Privacy Policy
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Administrators should also review the wording of the new default Privacy Policy static page. This can be found in |gears| "Admin > Static Pages >Privacy Policy", or it can be accessed via the Quick links menu in the AtoM header. Click on "edit" at the bottom of the view page, make your desired edits, and select "save" to save your changes or "cancel" to return to the default text.
+
+Upgrades to 2.4.1
+----------------------
+
+After first upgrading the cookie notification banner will appear as follows:
+
+.. image:: images/gdpr-banner-pre-task.*
+   :align: center
+   :width: 80%
+   :alt: GDPR banner first access
+
+* Additionally, if no further action is taken, the new default Privacy policy
+  static page will not exist
+* If you don't intend to use the banner or need the Privacy policy static page,
+  you can disable the notification banner via |gears| *Admin > Settings > Privacy policy*
+* To create the Privacy policy static page and update the banner message, a
+  system administrator should run the following task from the root AtoM installation directory: ``php symfony tools:run lib/task/tools/addGdprSettings.php``
+
+Custom theme updates
+--------------------
+
+* If you have a custom theme and the file <code>scaffolding.less</code> has
```

Comment:
Looks like you've cut and pasted this from the wiki, but <code> elements might not work properly here. Instead you should just use two back ticks around content to escape it in-line.